test/babashka/nrepl/server_test.clj

```diff
@@ -336,10 +336,33 @@
             (is (= "variable" type))))
         (testing "eldoc of invalid characters"
           (bencode/write-bencode os {"op" "eldoc" "ns" "user"
-                                     "sym" "
"
+                                     "sym" "\r"
```

Comment:
somehow Calva (?) messed up the original value that github shows as ^M

I guess ^M is supposed to be "Carriage return", thus "\r" should be a safe equivalent.